src/backends/torch.c

```diff
@@ -137,6 +137,17 @@ RAI_Script *RAI_ScriptCreateTorch(RAI_Device device, int64_t deviceid, const cha
   ret->device = device;
   ret->deviceid = deviceid;
   ret->refCount = 1;
+  if (device == RAI_DEVICE_CPU){
```

Comment:
See my more general comment below.

Reply:
@lantiga done. please revise the changes on all backends for model_create and on torch for script_create